src/OrchardCore.Modules/OrchardCore.Contents/Views/Contents-Button-Actions.SummaryAdmin.cshtml

```diff
@@ -5,34 +5,37 @@
     ContentItem contentItem = Model.ContentItem;
     var hasPublished = await ContentManager.HasPublishedVersionAsync(contentItem);
     var hasDraft = contentItem.HasDraft();
-    var hasClonePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.CloneContent, contentItem);
-    var hasDeletePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.DeleteContent, contentItem);
-    var hasEditPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.EditContent, contentItem);
-    var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PreviewContent, contentItem);
-    var hasPublishPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PublishContent, contentItem);
+    var hasDeletePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.CommonPermissions.DeleteContent, contentItem);
```

Comment:
The common permissions are assigned = to the modules permission statically, so it doesn't matter which one we check.

It's possible we could (maybe another pr) just remove the modules permissions, to get rid of any confusion there.

Reply:
Yes they are the same. But we should be using the CommonPermissions as they are shared globally. Here in the OrchardCore.Contents module it doesn't really matter but I think that we should still be using CommonPermissions for keeping "consistency" throughout the app.

Reply:
Done